plugins/aws/redshift/redshiftNodesCount.js

```diff
@@ -0,0 +1,69 @@
+var async = require('async');
+var helpers = require('../../../helpers/aws');
+
+module.exports = {
+    title: 'Redshift Nodes Count',
+    category: 'Redshift',
+    description: 'Ensures that your AWS account has not reached the limit set for the number of Redshift cluster nodes.',
+    more_info: 'The number of provisioned Amazon Redshift cluster nodes must be less than the provided nodes limit to avoid reaching the limit and exceeding the set budget.',
+    link: 'https://docs.aws.amazon.com/redshift/latest/mgmt/working-with-clusters.html#working-with-clusters-overview',
+    recommended_action: 'Remove Redshift clusters over defined limit',
+    apis: ['Redshift:describeClusters'],
+    settings: {
+        redshift_nodes_count: {
+            name: 'Amazon Redshift Nodes Count',
+            description: 'Maximum Amazon Redshift nodes count per region',
+            regex: '^.*$',
+            default: 'false'
```

Comment:
by making the default false, it makes it seem like a boolean field. this seems like an integer so it should be 0

Reply:
Right. And I think as this is denoting max limit, so it should be 0 as well. So I have set it to 5 by default. Let me know if you want me to change it